yaml/landuse.yaml

```diff
@@ -891,14 +891,13 @@ filters:
       <<: *output_properties
       kind: groyne
       tier: 4
-  # dike
+  # dyke
   - filter:
-      man_made: dike
-      geom_type: polygon
-    min_zoom: { max: [ 13, *tier4_min_zoom ] }
```

Comment:
Why is the `min_zoom` changing here?

Reply:
This probably needs same treatment as `dam` where polygons and lines are treated differently.